src/config.rs

```diff
@@ -413,7 +413,7 @@ fn get_palette<'a>(
     if palette.is_some() {
         log::trace!("Found color palette: {}", palette_name);
     } else {
-        log::debug!("Could not find color palette: {}", palette_name);
+        log::warn!("Could not find color palette: {}", palette_name);
```

Comment:
This will print a warning with the default starship config. (No palette defined and palette_name is an empty string). This should ignore empty strings or use `Option<&str>`.

The warning is also a bit noisy and will print multiple times depending on the configuration.